protocol/daemons/pricefeed/client/client.go

```diff
@@ -49,8 +49,14 @@ type Client struct {
 
 	// Ensure stop only executes one time.
 	stopDaemon sync.Once
+
+	// include HealthCheckableImpl to track the health of the daemon.
+	daemontypes.HealthCheckableImpl
```

Comment:
nit: move this to the top of `Client` so it's clear that this struct implements a client / has an anonymous field